packages/sit-onyx/src/components/OnyxDataGrid/features/HeaderCell.vue

```diff
@@ -28,6 +28,7 @@ const slots = defineSlots<{
 
   &__actions {
     display: inline-flex;
+    gap: inherit;
```

Comment:
```suggestion
    gap: var(--onyx-density-2xs);
```

Also the parent header gap should be changed to `var(--onyx-density-xs)` instead of `var(--onyx-spacing-2xs)` to align with Figma :)